cmd/finch/nerdctl_native.go

```diff
@@ -92,3 +89,17 @@ var osAliasMap = map[string]string{}
 var osArgHandlerMap = map[string]map[string]argHandler{}
 
 var osCommandHandlerMap = map[string]commandHandler{}
+
+func mapToString(m map[string]string) string {
```

Comment:
Can de-dupe this code too? Not a priority and can be done in a followup PR.

Reply:
+1 Agreed!

In another iteration, it could be possible to completely refactor all of the handlers into the generalized level with OS-specific handlerHelpers and simplifying the handlerMap aggregation.
This de-dupe and optimization can be included then.